### DISCUSSION TITLE
Build script

### DISCUSSION THREAD

**Discussion by ekkis** [vercel/vercel#4262] [General]
the [FAQ](https://vercel.com/docs/v2/platform/frequently-asked-questions) indicates that my `package.json` should contain:
```json
{
  "scripts": {
    "build": "[my-framework] build --output public"
  }
}
```
but doesn't give me any clue as to what `[my-framework]` is.  in my case, I just have an `index.js` that I want to publish.  without a `now.json` (which is what I used to use) when I visit the project after publishing I just get the text of the .js

where are intelligible instructions I can follow?

**Comment by coetry** [comment]
The build script snippet is just giving a general idea that frameworks usually have some sort of build command and the result of that build needs to output to the `public` directory in order for us to serve it.

> I just have an index.js that I want to publish

Is it a script that generates static files to be served?

**Reply by ekkis** [reply]
the index.js contains my website

**Comment by styfle** [comment]
>  in my case, I just have an index.js that I want to publish

It sound like you want a single API function.
You could rename `/index.js` to `/api/index.js`, then add a `/public/index.html` for the static frontend.
